lib/nodejs/buffered-worker-pool.js

```diff
@@ -75,7 +75,23 @@ class BufferedWorkerPool {
       process.execArgv.join(' ')
     );
 
-    this.options = {...WORKER_POOL_DEFAULT_OPTS, opts, maxWorkers};
+    let counter = 0;
+    const onCreateWorker = ({forkOpts}) => {
+      return {
+        forkOpts: {
+          ...forkOpts,
```

Comment:
I don't see where `process.env` is passed before this PR. Is it part of `WORKER_POOL_DEFAULT_OPTS` (as default without listing it)?
If you add more environmant variables, are they still passed to the child-processes?

Reply:
ok, tested.